app/styles/mixins.less

```diff
@@ -74,6 +74,39 @@
   margin-right: 8px;
 }
 
+// Margins
+.margin-top() {
+  margin-top: 8px;
+}
```

Comment:
I'm wondering if it might make sense to do this with variables and a little math:

```less
.margin-top(@x: 1) {
  margin-top: (@x * 1)px;
}
```
(not sure if that syntax is exactly right, but should be possible).
The advantages:
- Is a single mixin to remember
- Enforces only applying in 8 pixel increments

Reply:
yeah definitely will switch to this after this PR